bindings/go/oci/integration/integration_test.go

```diff
@@ -181,6 +185,12 @@ func Test_Integration_OCIRepository(t *testing.T) {
 			uploadDownloadBarebonesComponentVersion(t, repo, "test-component", "v1.0.0")
 		})
 
+		t.Run("basic upload and download of a component version", func(t *testing.T) {
+			repo, err := oci.NewRepository(oci.WithResolver(resolver), oci.WithReferrerTrackingPolicy(oci.ReferrerTrackingPolicyByIndexAndSubject))
+			r.NoError(err)
+			uploadDownloadBarebonesComponentVersion(t, repo, "test-component", "v1.0.0")
+		})
```

Comment:
Basically testing that it's still the same thing?  Should this be asserting something else? The name of it is definitely the same.

Reply:
I agree, will add the referrer policy to the test